### ISSUE TITLE
[Task]: It can be detected when the feature points occupy a certain proportion of the original image？

### ISSUE THREAD

**Issue by xingfenghaizeiwang** [openvinotoolkit/anomalib#1315]
### What is the motivation for this task?

I want to detect missing characters on the label

### Describe the solution you'd like

dataset:
  name: mvtec
  format: folder
  root: D:\project\anomalib\datasets\MVTec
  normal_dir: card/train/good
  normal_test_dir: card/test/good
  abnormal_dir: card/test/ng
  task: segmentation
  mask_dir: null
  extensions: null
  train_batch_size: 32
  eval_batch_size: 32
  num_workers: 8
  image_size:
  - 256
  - 256
  center_crop: null
  normalization: imagenet
  transform_config:
    train: null
    eval: null
  test_split_mode: none
  test_split_ratio: 0.2
  val_split_mode: synthetic
  val_split_ratio: 0.5
  tiling:
    apply: false
    tile_size: null
    stride: null
    remove_border_count: 0
    use_random_tiling: false
    random_tile_count: 16
model:
  name: padim
  backbone: wide_resnet50_2
  pre_trained: true
  layers:
  - layer1
  - layer2
  - layer3
  normalization_method: min_max
  input_size:
  - 256
  - 256
metrics:
  image:
  - F1Score
  - AUROC
  pixel:
  - F1Score
  - AUROC
  threshold:
    method: adaptive
    manual_image: null
    manual_pixel: null
visualization:
  show_images: false
  save_images: true
  log_images: true
  image_save_path: null
  mode: full
project:
  seed: 42
  path: result\padim\mvtec\run
  unique_dir: false
logging:
  logger: []
  log_graph: false
optimization:
  export_mode: onnx
trainer:
  enable_checkpointing: true
  default_root_dir: result\padim\mvtec\run
  gradient_clip_val: 0
  gradient_clip_algorithm: norm
  num_nodes: 1
  devices: 1
  enable_progress_bar: true
  overfit_batches: 0.0
  track_grad_norm: -1
  check_val_every_n_epoch: 1
  fast_dev_run: false
  accumulate_grad_batches: 1
  max_epochs: 1
  min_epochs: null
  max_steps: -1
  min_steps: null
  max_time: null
  limit_train_batches: 1.0
  limit_val_batches: 1.0
  limit_test_batches: 1.0
  limit_predict_batches: 1.0
  val_check_interval: 1.0
  log_every_n_steps: 50
  accelerator: auto
  strategy: null
  sync_batchnorm: false
  precision: 32
  enable_model_summary: true
  num_sanity_val_steps: 0
  profiler: null
  benchmark: false
  deterministic: false
  reload_dataloaders_every_n_epochs: 0
  auto_lr_find: false

  replace_sampler_ddp: true
  detect_anomaly: false
  auto_scale_batch_size: false
  plugins: null
  move_metrics_to_cpu: true
  multiple_trainloader_mode: max_size_cycle

![1](https://github.com/openvinotoolkit/anomalib/assets/104884533/29e3a6ff-2873-4100-a52f-887a8000ec10)
![4](https://github.com/openvinotoolkit/anomalib/assets/104884533/eb806084-06a8-4fec-b953-4ceede4e74ef)
![2](https://github.com/openvinotoolkit/anomalib/assets/104884533/b42be417-d2cb-4385-bd49-5e6ee4e99f23)

**Comment by blaz-r** [comment]
Hello. Could you clarify this issue please? The title and request don't match so I'm not sure what is the problem.

**Comment by samet-akcay** [comment]
I agree with @blaz-r. @xingfenghaizeiwang, could you please clarify the issue you are having?

**Comment by xingfenghaizeiwang** [comment]
> 你好。你能澄清一下这个问题吗？标题和请求不匹配，所以我不确定问题出在哪里。

Unable to identify the characters on the card indeed

**Comment by xingfenghaizeiwang** [comment]
> 我同意.，您能否澄清您遇到的问题？

Unable to identify the characters on the card indeed.I'm not sure how small a target can be identified
